sycl/include/CL/sycl/item.hpp

```diff
@@ -86,6 +88,7 @@ template <int dimensions = 1, bool with_offset = true> struct item {
   // For call constructor inside conversion operator
   friend struct item<dimensions, false>;
```

Comment:
Probably everything along is a `class` instead of a `struct` now

Reply:
item is still a struct

Reply:
It is always good to have some feedback from the implementors.
If you find some discrepancies in the spec, feel free to file an issue or even better a PR :-) on https://github.com/KhronosGroup/SYCL-Docs
Note that the published PDF might be out-dated compared to the source files.